arch/risc-v/src/mpfs/mpfs_usb.c

```diff
@@ -129,6 +129,10 @@
 #  define MSB 1
 #endif
 
+#ifndef MPFS_USB_DMA_ADDR_UPPER_OFFSET
+#  define MPFS_USB_DMA_ADDR_UPPER_OFFSET 0x14u
```

Comment:
Extra space

Reply:
These are according to the nuttx coding standard, indentation here is correct

Reply:
Done similarly it has been done earlier in that file.